pubsub/subscriptions/subscription_test.go

```diff
@@ -155,6 +158,69 @@ func TestIAM(t *testing.T) {
 	})
 }
 
+func TestPullMsgsSync(t *testing.T) {
+	ctx := context.Background()
+	tc := testutil.SystemTest(t)
+	client := setup(t)
+
+	topic := client.Topic(topicName)
+
+	// Publish 10 messages on the topic.
+	const numMessages = 10
+	var results []*pubsub.PublishResult
+	for i := 0; i < numMessages; i++ {
+		res := topic.Publish(ctx, &pubsub.Message{
+			Data: []byte(fmt.Sprintf("hello world #%d", i)),
+		})
+		results = append(results, res)
+	}
+
+	// Check that all messages were published.
+	for _, r := range results {
+		if _, err := r.Get(ctx); err != nil {
+			t.Fatalf("Get publish result: %v", err)
+		}
+	}
+
+	buf := new(bytes.Buffer)
+	err := pullMsgsSync(buf, tc.ProjectID, subName, topic)
+	if err != nil {
+		t.Fatalf("failed to pull messages: %v", err)
+	}
+	// Check for number of newlines, which should correspond with number of messages.
+	got := strings.Count(buf.String(), "\n")
+	if got != numMessages {
+		t.Fatalf("got %d messages, want %d", got, numMessages)
```

Comment:
```suggestion
		t.Fatalf("pullMsgsSync got %d messages, want %d", got, numMessages)
```

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures